pkg/controller/awsprivatelink/awsprivatelink_controller.go

```diff
@@ -1186,7 +1186,7 @@ func initialURL(c client.Client, key client.ObjectKey) (string, error) {
 		key,
 		kubeconfigSecret,
 	); err != nil {
-		return "", errors.Wrap(err, "could not get admin kubeconfig secret")
+		return "", err
```

Comment:
the apierrors.IsNotFound supports wrapped errors, so i don't think we need to *not* wrap the error here.
https://github.com/kubernetes/apimachinery/blob/e337f44144a6ffeee2758c80dea17dd53fa6cbf4/pkg/api/errors/errors.go#L650
https://github.com/kubernetes/apimachinery/blob/e337f44144a6ffeee2758c80dea17dd53fa6cbf4/pkg/api/errors/errors.go#L480-L482

Reply:
Cool that was not what I would have assumed.